### DISCUSSION TITLE
유니티 에디터 확장을 해도 퍼블리싱할때 문제가 없을까요?

### DISCUSSION THREAD

**Discussion by pda017** [naverz/zepeto-studio-kor#844] [5. Q&A]
cs 파일을 만들기만 해도 퍼블리싱할때 에러가 뜨는것 같던데
에디터 확장쪽 스크립트도 쓸수 없는건가요?
아니면 Editor 폴더는 제외한다던가 그런게 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, Editor 관련내용들은 런타임에 반영되지 않으므로 기존 유니티와 동일하게 C#으로 제작하고 사용하실 수 있습니다.
